cs-amazon/src/main/java/io/cloudslang/content/amazon/entities/constants/Constants.java

```diff
@@ -177,4 +177,12 @@
     public static class S3QueryApiActions {
         public static final String GET_BUCKET = "GET Bucket";
     }
+
+    public static class ServiceCatalogActions {
+        public static final String CREATE_IN_PROGRESS = "CREATE_IN_PROGRESS";
+        public static final String CREATE_COMPLETE = "CREATE_COMPLETE";
+        public static final String FAILED = "FAILED";
+        public static final String CLOUD_FORMATION_STACK_NAME_REGEX = "(SC)-[0-9]{0,63}-[a-z]{0,63}-[a-z0-9]{0,63}";

```

Comment:
Each ARN identifies the resource by specifying the stack then the name. Can we use a simpler reg-ex to avoid cases when AWS is changing the format of the name?

arn:aws:cloudformation:us-east-1:981593490063:**stack**/SC-981593490063-pp-45gnaw5zmcuv2/9f413790-aaa4-11e8-b5e7-50fae984a035